Source/bmalloc/libpas/src/libpas/pas_config.h

```diff
@@ -28,6 +28,21 @@
 
 #include "pas_config_prefix.h"
 
+#include "stdbool.h"
+
+#define PAS_LOG_NONE (0)
+#define PAS_LOG_HEAP_INFRASTRUCTURE (1 << 0)
+#define PAS_LOG_BOOTSTRAP_HEAPS (1 << 1)
+#define PAS_LOG_IMMORTAL_HEAPS (1 << 2)
+#define PAS_LOG_SEGREGATED_HEAPS (1 << 3)
+#define PAS_LOG_BITFIT_HEAPS (1 << 4)
+#define PAS_LOG_LARGE_HEAPS (1 << 5)
+#define PAS_LOG_JIT_HEAPS (1 << 6)
+#define PAS_LOG_SURFACE_ALLOCATIONS (1 << 7)  /* Heap-type agnostic */
```

Comment:
nit: It feels like the term SURFACE sort of implies something.  How about naming this as something like PAS_LOG_OTHER_ALLOCATIONS or PAS_LOG_HEAP_TYPE_AGNOSTIC_ALLOCATIONS or just PAS_LOG_OTHER  instead?  I prefer PAS_LOG_OTHER as it's short, most generic, and applies to all logging, not just allocations ... and you do seem to be using it that way.